nipy/core/image/image_spaces.py

```diff
@@ -60,7 +60,7 @@
 It also works with nibabel images, which can only have xyz_affines:
 
 >>> import nibabel as nib
->>> nimg = nib.Nifti1Image(data, affine)
+>>> nimg = nib.Nifti1Image(data, affine, dtype=data.dtype)
```

Comment:
This will set a hard minimum of `nibabel >= 4.0`. The alternative is to set a non-int64 dtype on data (L21).